subjects/countstars/README.md

```diff
@@ -0,0 +1,43 @@
+## CountStars
+
+### Instructions
+Write a function called CountStars that counts the stars up to a number given as an argument.
+- If the number is negative or equal to 0, return "`No stars`"
+- No need to manage overflow.
+
+The Library strconv is allowed.
+
+### Expected Function
+
+```go
+func CountStars(num int) string {
+
+}
+```
+### Usage
+
+Here is a possible program to test your function :
+
+```go
```

Comment:
Fix this example.
The imports are wrong, its missing the package ...

Reply:
Done